sop/modules/OSD/partials/ocs_extend_cluster.adoc

```diff
@@ -0,0 +1,27 @@
+
+To Extend the cluster , and reset is back, follow the below mentioned steps
```

Comment:
What is `reset is back` here ?

Reply:
Also we are not extending the cluster, we are extending the cluster full threshold